src/OrchardCore/OrchardCore.ContentLocalization.Abstractions/Records/LocalizedContentItemIndex.cs

```diff
@@ -101,7 +101,7 @@ public void Describe(DescribeContext<ContentItem> context)
                     return new LocalizedContentItemIndex
                     {
                         Culture = !partRemoved ? part.Culture.ToLowerInvariant() : null,
```

Comment:
We can reduce the number of checks to one

```csharp
if (partRemoved)
{
    Culture = LocalizationSet = null;
}
else
{
    Culture = part.Culture.ToLowerInvariant();
    LocalizationSet = part.LocalizationSet;
}
```

Reply:
This is inside an object initializer so we can't do that. If we want to reduce the number of `if`s then from line 101 this has to be broken up with an object initializer, then conditional assignment, then `return` part, which looks a lot messier.

Reply:
Oops, I saw the object initializer now ;) forget my comment

Reply:
No worries! Thanks for reviewing.